api/sys_plugins.go

```diff
@@ -345,28 +345,29 @@ func (c *Sys) ReloadPluginStatusWithContext(ctx context.Context, reloadStatusInp
 	if err != nil {
 		return nil, err
 	}
+	if resp == nil {
+		return nil, errors.New("nil plugin status response")
+	}
+
 	defer resp.Body.Close()
```

Comment:
defer the closing of the body even if the response is nil
```suggestion
    defer resp.Body.Close()
    
	if resp == nil {
		return nil, errors.New("nil plugin status response")
	}
```